### PR TITLE
Add roxdb workflow script

### PR DESCRIPTION
**Pull request by dcaravel**
Adds a `roxdb` workflow script that allows for quickly connecting to `central-db` (default) or `scanner-db`.

Examples:
```sh
$ roxdb
psql (13.12)
Type "help" for help.

central_active=# 
```

```sh
$ roxdb scanner
psql (12.16)
Type "help" for help.

postgres=# 
```

### CODE REVIEW

**Review by dashrews78 — APPROVED** [review]
LGTM.  This is one of those that make me mad at myself for not doing this a year ago.

File: scripts/runtime/roxdb.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+# Example usages:
+#   roxdb central        Connects to central-db
+#   roxdb scanner        Connects to scanner-db
+
+SCRIPT="$(python3 -c 'import os, sys; print(os.path.realpath(sys.argv[1]))' "${BASH_SOURCE[0]}")"
+source "$(dirname "$SCRIPT")/../../lib/common.sh"
+
+usage() {
+  die "Usage: $0 [central/scanner] [namespace]"
```
Comment (RTann) [inline scripts/runtime/roxdb.sh:16]: Do we use this namespace parameter?

Comment (dcaravel) [reply scripts/runtime/roxdb.sh:16]: nice catch, was adding it, then removed it it, will remove

Comment (dcaravel) [reply scripts/runtime/roxdb.sh]: https://github.com/stackrox/workflow/pull/102/commits/0a75233d218ffc0f49d004b7315d9fcbb159ed72

**Review by RTann — APPROVED** [review]
LGTM though now I'm curious why we don't let people specify a namespace

**Comment by dcaravel** [comment]
> LGTM though now I'm curious why we don't let people specify a namespace

I didn't add because I was boxing the time I spent working on this, can add it in a latter PR.

**Pull request merged**